haystack/components/retrievers/sentence_window_retriever.py

```diff
@@ -134,7 +134,9 @@ def run(self, retrieved_documents: List[Document]):
         :param retrieved_documents: List of retrieved documents from the previous retriever.
         :returns:
             A dictionary with the following keys:
-            - `context_windows`:  List of strings representing the context windows of the retrieved documents.
+            - `context_windows`:  Strings representing the context windows text of the retrieved documents.
+            - `context_documents`: The retrieved context window documents.
```

Comment:
Initially, I found it hard to understand why the returned documents are a list of lists instead of a flat list (also because it differs from the PR description).

I think the description of `context_documents` could be made more expressive.

Reply:
improved it a bit, but if you still don't agree, feel free to commit your own suggestion

Reply:
I think that something similar to this would be clearer:
```python
- `context_windows`: 
    A list of strings, where each string represents the concatenated text from the context window of the 
    corresponding document in `retrieved_documents`.
- `context_documents`: 
    A list of lists of `Document` objects, where each inner list contains the documents that form the 
    context window for the corresponding document in `retrieved_documents`.
```